### PR TITLE
F11 to toggle fullscreen

### PR DESCRIPTION
**Pull request by TomerZeitune**
Hey, I added F11 default keybinding to toggle fullscreen mode.

### CODE REVIEW

**Comment by skipmeister123** [comment]
Why were we lacking this in the first place? 😅 Thank you!

**Comment by felixse** [comment]
@mjs whats your opinion on this, regarding #309?

**Comment by mjs** [comment]
It's probably OK. gnome-terminal uses F11 for full screen too, so this key is probably not widely used by console apps.

File: FluentTerminal.App.Services/Implementation/DefaultValueProvider.cs [hunk 1]
```diff
@@ -259,6 +259,14 @@ public ICollection<KeyBinding> GetDefaultKeyBindings(Command command)
                         Alt = true,
                         Shift = false,
                         Key = (int)ExtendedVirtualKey.Enter
+                    },
+                      new KeyBinding
+                    {
+                        Command = nameof(Command.ToggleFullScreen),
+                        Ctrl = false,
+                        Alt = false,
+                        Shift = false,
```
Comment (mjs) [inline FluentTerminal.App.Services/Implementation/DefaultValueProvider.cs]: There's no need to put in the `false` values. This is the default if not specified and it's easier to read the definition without them.

**Comment by felixse** [comment]
Thanks 👍

**Pull request merged**